recipe/libgfortran.spec

```diff
@@ -10,4 +10,3 @@
 *libgcc:
 %{static-libgcc|static:                                                          %{!m32:%:version-compare(>= 10.6 mmacosx-version-min= -lSystem)}                  -lgcc_eh -lgcc;                                                             shared-libgcc|fexceptions|fgnu-runtime:                                          %:version-compare(!> 10.5 mmacosx-version-min= -lSystem)                  %:version-compare(>< 10.5 10.6 mmacosx-version-min= -lSystem)          %:version-compare(!> 10.5 mmacosx-version-min= -lSystem)             %:version-compare(>= 10.5 mmacosx-version-min= -lSystem)                  -lgcc ;                                     :%:version-compare(>< 10.3.9 10.5 mmacosx-version-min= -lSystem)        %:version-compare(>< 10.5 10.6 mmacosx-version-min= -lSystem)          %:version-compare(!> 10.5 mmacosx-version-min= -lSystem)           %:version-compare(>= 10.5 mmacosx-version-min= -lSystem)                  -lgcc }
```

Comment:
Does this line need a closing `}`? I have no idea what is supposed to be in a spec file.

Reply:
there are 2 opening brackets and 2 closing ones

Reply:
ohhhh I missed it! sorry for the noise!